eras/alonzo/test-suite/test/Test/Cardano/Ledger/Alonzo/Golden.hs

```diff
@@ -17,21 +17,24 @@ import Cardano.Ledger.Alonzo.PParams (
   LangDepView (..),
   getLanguageView,
  )
-import Cardano.Ledger.Alonzo.Scripts (
-  CostModel,
-  CostModels (..),
-  ExUnits (..),
-  Prices (..),
-  mkCostModel,
- )
 import Cardano.Ledger.Alonzo.TxBody (AlonzoTxOut (..), utxoEntrySize)
 import Cardano.Ledger.BaseTypes (StrictMaybe (..), boundRational)
 import Cardano.Ledger.Binary (decCBOR, decodeFullAnnotator)
 import Cardano.Ledger.Binary.Plain as Plain (serialize)
 import Cardano.Ledger.Block (Block (..))
 import Cardano.Ledger.Coin (Coin (..))
 import Cardano.Ledger.Mary.Value (valueFromList)
+import Cardano.Ledger.Plutus.CostModels (
```

Comment:
Shouldn't we update the dependency on cardano-ledger-core  in [cardano-ledger-alonzo-test.cabal ](https://github.com/IntersectMBO/cardano-ledger/blob/lehins/cost-models-fixes/eras/alonzo/test-suite/cardano-ledger-alonzo-test.cabal#L49) to 

` cardano-ledger-core:{cardano-ledger-core, testlib} >=1.10 && <1.11`  ?

Reply:
Yes. Nice catch. Done.